### PR TITLE
Update deps.

### PR DESCRIPTION
**Pull request by howbazaar**
Use collections/set not utils/set.

### CODE REVIEW

File: dependencies.tsv [hunk 1]
```diff
@@ -1,11 +1,13 @@
+github.com/juju/collections	git	520e0549d51ae2b50f44f4df2b145a780a5bc6e0	2018-05-15T20:37:31Z
 github.com/juju/errors	git	1b5e39b83d1835fa480e0c2ddefb040ee82d58b3	2015-09-16T12:56:42Z
-github.com/juju/loggo	git	8477fc936adf0e382d680310047ca27e128a309a	2015-05-27T03:58:39Z
-github.com/juju/names	git	8a0aa0963bbacdc790914892e9ff942e94d6f795	2016-03-30T15:05:33Z
```
Comment (Veebers) [inline dependencies.tsv:3]: is juju/names no longer used?

**Review by Veebers — COMMENTED** [review]
One question otherwise LGTM (quite straight forward).

Probably worth getting a merge/check job setup for this repo?

File: dependencies.tsv (hunk 1 above)
Comment (howbazaar) [reply dependencies.tsv:3]: Nope, old and gone.

**Comment by howbazaar** [comment]
!!build!!

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**